### ISSUE TITLE
Misplaced class in diagram

### ISSUE THREAD

**Issue by pato-p** [mjbvz/vscode-markdown-mermaid#43]
When I have multiple class diagrams in a single document, some classes are misplaced. There is no issue if there is only one class diagram.

In this example, the class `Misplaced` is rendered behind the class `A`, and the arrow from `A` points to a void space:
~~~
``` mermaid
classDiagram
class Misplaced
class A
A --> Misplaced
```
``` mermaid
classDiagram
class X
```
~~~

Output:
![test](https://user-images.githubusercontent.com/16959272/68921490-3f244400-0779-11ea-9bdb-bbf5d7298a9b.png)

**Comment by mjbvz** [comment]
Is this VS Code specific? Do you see the same if you use the mermaid cli?

**Comment by pato-p** [comment]
With Mermaid CLI I cannot generate SVG for both diagrams at once, so I believe there is no issue there. The thing here is that just the presence of the second diagram in Markdown document affects the rendering of the first diagram. If I delete the second diagram, the first diagram renders as expected.

**Comment by seflue** [comment]
I can confirm this problem. Multiple class diagrams in one markdown document are not properly supportet. If I declare a second diagram, I get misplaced classes.

**Comment by landru29** [comment]
Dependency to mermaid must be changed to `8.4.3` in `package.json`

Workaround:
```bash
cd $HOME/.vscode/extensions
git clone https://github.com/mjbvz/vscode-markdown-mermaid.git
rm -rf bierner.markdown-mermaid-1.4.1
cd vscode-markdown-mermaid
```
in Package.json change `8.4.0` to `8.4.3`

```bash
npm install
cd ..
mv vscode-markdown-mermaid bierner.markdown-mermaid-1.4.1
```

**Comment by mjbvz** [comment]
Want to submit a PR that bumps the mermaid version?

**Comment by AceLondon** [comment]
@mjbvz This seems to be fixed with the upgrade to mermaid 8.4.6

**Comment by pato-p** [comment]
Yes, I can confirm that this issue is fixed, thanks!
